gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_2/00_add_event_organizations.yml

```diff
@@ -13,6 +13,14 @@ databaseChangeLog:
             columns:
```

Comment:
`addPrimaryKey` detected in liquibase changelog 💥. Please create the primary key directly with the create table statement.